### PR TITLE
Add and fix filed Symbol struct futures exchangeInfoService and add SetApiEndpoint

### PR DESCRIPTION
**Pull request by mmavka**
add:
- LiquidationFee string \`json:"liquidationFee"\`
- MarketTakeBound string \`json:"marketTakeBound"\`

fix:
- OrderType []OrderType \`json:"orderType"\`

### CODE REVIEW

File: v2/client.go [hunk 1]
```diff
@@ -423,6 +423,12 @@ func (c *Client) callAPI(ctx context.Context, r *request, opts ...RequestOption)
 	return data, nil
 }
 
+// SetApiEndpoint set api Endpoint
+func (c *Client) SetApiEndpoint(url string) *Client {
+	c.BaseURL = url
```
Comment (adshao) [inline v2/client.go:428]: Why not set API endpoint with `c.BaseURL = url`?

**Review by adshao — CHANGES_REQUESTED** [review]

File: v2/client.go (hunk 1 above)
Comment (mmavka) [reply v2/client.go:428]: one string futures.NewClient("", "").SetApiEndpoint("...")

File: v2/futures/exchange_info_service.go [hunk 2]
```diff
@@ -67,12 +67,14 @@ type Symbol struct {
 	UnderlyingSubType     []string                 `json:"underlyingSubType"`
 	SettlePlan            int64                    `json:"settlePlan"`
 	TriggerProtect        string                   `json:"triggerProtect"`
-	OrderType             []OrderType              `json:"OrderType"`
+	OrderType             []OrderType              `json:"orderType"`
```
Comment (adshao) [inline v2/futures/exchange_info_service.go:70]: could you add unit test for these changes?

Comment (mmavka) [reply v2/futures/exchange_info_service.go:70]: Yep. Done

**Review by adshao — APPROVED** [review]

**Comment by mmavka** [comment]
websocket_service_test... I not edit. If not need help...

**Pull request merged**